bears/python/PEP8Bear.py

```diff
@@ -31,14 +31,19 @@ def run(self, filename, file,
         Detects and fixes PEP8 incompliant code. This bear will not change
         functionality of the code in any way.
 
-        :param max_line_length:   Maximum number of characters for a line.
-                                  When set to 0 allows infinite line length.
-        :param indent_size:       Number of spaces per indentation level.
-        :param pep_ignore:        A list of errors/warnings to ignore.
-        :param pep_select:        A list of errors/warnings to exclusively
-                                  apply.
-        :param local_pep8_config: Set to true if autopep8 should use a config
-                                  file as if run normally from this directory.
+        :param max_line_length:
+            Maximum number of characters for a line
+            When set to 0 allows infinite line length.
+        :param indent_size:
+            Number of spaces per indentation level.
+        :param pep_ignore:
+            A list of errors/warnings to ignore.
+        :param pep_select:
+            A list of errors/warnings to exclusively
+            apply.
```

Comment:
This one also.

Reply:
https://github.com/coala/coala-bears/pull/2701#discussion_r226402197

Reply:
this one ... ewww ... put it all on the one line.